src/Pages/HumansAtNasa/HumansAtNasa.tsx

```diff
@@ -9,9 +9,24 @@ export function HumansAtNasa() {
             <h1 className='opportunity'>Humans At Nasa</h1>
             <p>NASA's Mars missions are designed, built and operated by some of the best and brightest. Meet some of the amazing people who do their best work on another planet.</p>
             <HumansInSpace/>
-            <HumanAtNasa name={"Laura Kerber"} jobTitle={"Research Scientist"} jobDescription={"Laura Kerber is a research scientist at JPL with interests in physical volcanology, aeolian geomorphology, wind over complex surfaces, the ancient Martian climate."} imageUrl={"https://science.jpl.nasa.gov/images/people-3979.jpg"}/>
-            <HumanAtNasa name={"Dr. Timothy Parker"} jobTitle={"Geology Team Group Lead"} jobDescription={"Dr. Tim Parker was the originator of the Mars Ocean hypothesis. He earned his Ph.D. at the University of Southern California in 1994, with a dissertation entitled \"Martian Paleolakes and Oceans.\""} imageUrl={"https://science.jpl.nasa.gov/images/people-135.jpg"}/>
-            <HumanAtNasa name={"Nathan Williams"} jobTitle={"Planetary Geologist"} jobDescription={"Nathan Williams is a planetary geologist with particular interest in the remote sensing of planetary surfaces, especially relating to structural geology, geodesy, and landform evolution."} imageUrl={"https://science.jpl.nasa.gov/images/people-4770.jpg"}/>
+            <Photocard imageUrl={"https://science.jpl.nasa.gov/images/people-3979.jpg"}
+                       altText={"Laura Kerber"}
+                       imageTitle={"Laura Kerber - Research Scientist"}
+                       description={"Laura Kerber is a research scientist at JPL with interests in physical volcanology, aeolian geomorphology, wind over complex surfaces, the ancient Martian climate."}
+                       testid="photocard"
```

Comment:
We don't seem to use this id anywhere - do we actually need it?
If we do, wouldn't we want them all to be unique (rather than all being 'photocard')?

Reply:
Indeed however someone added this in to Photocards do tests on the Discover Rover Page, without checking here.

Reply:
I have added some more descriptive testIDs

Reply:
cool - fine for how then.

A better fix would be to make that an optional property on the Photocard rather than a required one.
That way we don't need to do anything if we don't want to!